vcd/datasource_vcd_storage_profile.go

```diff
@@ -32,6 +32,70 @@ func datasourceVcdStorageProfile() *schema.Resource {
 				Required:    true,
 				Description: "Name of storage profile",
 			},
+			"limit": &schema.Schema{
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "Maximum number of storage bytes (scaled by Units) allocated for this profile. A value of 0 is understood to mean `maximum possible`",
```

Comment:
A reference to `Units` field as reading it is hard to understands.

```suggestion
				Description: "Maximum number of storage bytes (scaled by 'units' field) allocated for this profile. A value of 0 is understood to mean `maximum possible`",
```